Firestore/third_party/FirestoreEncoder/FirestoreDecoder.swift

```diff
@@ -388,24 +388,24 @@ private struct _FirestoreKeyedDecodingContainer<K: CodingKey>: KeyedDecodingCont
   public func nestedContainer<NestedKey>(keyedBy _: NestedKey.Type,
                                          forKey key: Key) throws
     -> KeyedDecodingContainer<NestedKey> {
-    decoder.codingPath.append(key)
-    defer { self.decoder.codingPath.removeLast() }
+      decoder.codingPath.append(key)
+      defer { self.decoder.codingPath.removeLast() }
 
-    guard let value = self.container[key.stringValue] else {
-      throw DecodingError.valueNotFound(KeyedDecodingContainer<NestedKey>.self,
-                                        DecodingError.Context(codingPath: codingPath,
-                                                              debugDescription: "Cannot get nested keyed container -- no value found for key \"\(key.stringValue)\""))
-    }
+      guard let value = self.container[key.stringValue] else {
+        throw DecodingError.valueNotFound(KeyedDecodingContainer<NestedKey>.self,
+                                          DecodingError.Context(codingPath: codingPath,
+                                                                debugDescription: "Cannot get nested keyed container -- no value found for key \"\(key.stringValue)\""))
+      }
 
-    guard let dictionary = value as? [String: Any] else {
-      throw DecodingError
-        ._typeMismatch(at: codingPath, expectation: [String: Any].self, reality: value)
-    }
+      guard let dictionary = value as? [String: Any] else {
+        throw DecodingError
+          ._typeMismatch(at: codingPath, expectation: [String: Any].self, reality: value)
+      }
 
-    let container = _FirestoreKeyedDecodingContainer<NestedKey>(referencing: decoder,
-                                                                wrapping: dictionary)
-    return KeyedDecodingContainer(container)
-  }
+      let container = _FirestoreKeyedDecodingContainer<NestedKey>(referencing: decoder,
+                                                                  wrapping: dictionary)
+      return KeyedDecodingContainer(container)
+    }
```

Comment:
The indentation of this method is now two spaces too far. This seems to be the only case that's broken this way so I'm OK with moving forward if there's no obvious fix.

Reply:
Filed https://github.com/nicklockwood/SwiftFormat/issues/697